shared/oae/css/oae.components.css

```diff
@@ -194,6 +194,85 @@ i.fa.large:before {
 }
 
 
+/****************
+ ** SWITCH TAB **
+ ****************/
+
+.oae-switchtab-container {
+    text-align: center;
+}
+
+.oae-switchtab-container .oae-switchtab-control {
```

Comment:
If we leverage Bootstrap pill tabs as much as possible, some of these should probably become overrides of their style.